intro-Linux/ch_common_pitfalls.tex

```diff
@@ -103,36 +103,37 @@ \subsubsection{Copying files with WinSCP}
 \subsubsection{Permissions}
 
 \begin{prompt}
-%\shellcmd{ls -l $<$file$>$ \# File with correct permissions}%
--rwxr-xr-\strong{x} 1 %\userid% %\userid% 2983 Jan 30 09:13 $<$file$>$
-%\shellcmd{ls -l $<$file$>$ \# File without correct permissions}%
--rw-r--r-\strong{-} 1 %\userid% %\userid% 2983 Jan 30 09:13 $<$file$>$
+%\shellcmd{ls -l script.sh \# File with correct permissions}%
+-rwxr-xr-x 1 %\userid% %\userid% 2983 Jan 30 09:13 script.sh
+%\shellcmd{ls -l script.sh \# File without correct permissions}%
+-rw-r--r-- 1 %\userid% %\userid% 2983 Jan 30 09:13 script.sh
 \end{prompt}
 
 The script you want to submit needs have correct permissions.
-before submitting you can
+Before submitting it, you'll need to add execute permissions
+to make sure it can be executed:
 
 \begin{prompt}
 %\shellcmd{chmod +x script\_name.sh}%
 \end{prompt}
 
-to make sure it can be executed.
+
 
 \subsection{Help\!}
 
 If you stumble upon an error, don't panic! Read the error output, it might
-contain a clue as to what went wrong. You can copy the error message into google
+contain a clue as to what went wrong. You can copy the error message into Google
 (selecting a small part of the error without filenames).
```

Comment:
Also mention that using double quotes around the error can help with getting good search results, but letting Google consider the error as one whole thing rather than words glued together in random order.